app/src/main/java/org/fossasia/openevent/general/data/Network.kt

```diff
@@ -4,7 +4,7 @@ import android.content.Context
 import android.net.ConnectivityManager
 import org.fossasia.openevent.general.OpenEventGeneral
 
-class Network {
+object Network {
```

Comment:
why?

Reply:
> why?

For call function from another class, it must object like **Utills.kt** class.

Reply:
see how its done in login fragment view model